test/src/helper/mock/tendermintMessage.ts

```diff
@@ -0,0 +1,257 @@
+// Copyright 2018-2019 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+import { H256, U64 } from "codechain-primitives";
+import { EventEmitter } from "events";
+import { compressSync, uncompressSync } from "snappy";
+
+const RLP = require("rlp");
+
+export const Emitter = new EventEmitter();
+
+enum MessageType {
+    MESSAGE_ID_CONSENSUS_MESSAGE = 0x01,
+    MESSAGE_ID_PROPOSAL_BLOCK = 0x02,
+    MESSAGE_ID_STEP_STATE = 0x03,
+    MESSAGE_ID_REQUEST_MESSAGE = 0x04,
+    MESSAGE_ID_REQUEST_PROPOSAL = 0x05,
+}
+
+export enum Step {
+    Propose = 0,
+    Prevote = 1,
+    Precommit = 2,
+    Commit = 3,
+}
+
+interface VoteStep {
+    height: number,
+    view: number,
+    step: Step,
+}
+
+export interface ConsensusMessage {
+    type: "consensusmessage",
+    messages: Array<{
+        on: {
+            step: VoteStep,
+            blockHash: H256 | null,
+        },
+        signature: string,
+        signerIndex: number,
+    }>;
+}
+
+export interface ProposalBlock {
+    type: "proposalblock",
+    signature: string,
+    view: number,
+    message: Buffer,
+}
+
+export interface StepState {
+    type: "stepstate",
+    voteStep: VoteStep,
+    proposal: H256 | null,
+    lockView: number | null,
+    knownVotes: Buffer
+}
+
+export interface RequestMessage {
+    type: "requestmessage",
+    voteStep: VoteStep,
+    requestedVotes: Buffer,
+}
+
+export interface RequestProposal {
+    type: "requestproposal",
+    height: number,
+    view: number,
+}
+
+type MessageBody = ConsensusMessage | ProposalBlock | StepState | RequestMessage | RequestProposal;
+
+function readOptionalRlp<T>(bytes: [] | [Buffer], decoder: (b: Buffer) => T) {
+    if (bytes.length === 0) {
+        return null;
+    } else {
+        return decoder(bytes[0]);
+    }
+}
+
+function readUIntRLP(bytes: Buffer) {
```

Comment:
Would you change the function name? It is not related to the RLP.

Reply:
How about `readUIntBE`?